pkg/k8s/version/version.go

```diff
@@ -44,22 +44,22 @@ type ServerCapabilities struct {
 type cachedVersion struct {
 	mutex        lock.RWMutex
 	capabilities ServerCapabilities
-	version      *go_version.Version
+	version      go_version.Version
 }
 
 var (
 	cached = cachedVersion{}
 
-	patchConstraint        = versioncheck.MustCompile(">= 1.13.0")
-	updateStatusConstraint = versioncheck.MustCompile(">= 1.11.0")
+	patchConstraint        = versioncheck.MustVersion("1.13.0")
```

Comment:
Why is `MustVersion` used here vs. `MustCompile` in other files? Is this so we can do the comparison operations below?

Reply:
Fixed